docs/content/latest/explore/ysql-language-features/queries.md

```diff
@@ -253,60 +254,72 @@ INSERT INTO employees (name, department) VALUES
 ```
 
 ```sql
-SELECT DISTINCT name FROM employees ORDER BY name DESC;  
+SELECT DISTINCT name FROM employees ORDER BY name DESC;
 ```
 
 The following is the output produced by the preceding examples:
 
 ```
-name																
+name
 --------------------
 Random Dude
-Lucille Ball 				
-John Smith 					
-John Zimmerman	
+Lucille Ball
+John Smith
+John Zimmerman
 Jean Harlow
-Bette Davis 				
+Bette Davis
+```
+
+### Case Sensitivity
+
+Since YSQL is case-sensitive (that is, it treats the same characters differently depending on whether they are uppercase and lowercase), it is important to use correct cases when forming queries. For example, a table called `Employees` would not be recognized as an `employees` table, even if that is what you mean. That is, if your database contains an `employees` table and you want to query it using the following statement, the query execution will result in an error:
+
+```sql
+SELECT name FROM Employees;
```

Comment:
As far as I know this section is not quite correct.
Identifiers are normalized to lowercase unless they are quoted.
So, for example:

```
yugabyte=# create table fOo(iD int PRIMARY KEY);
CREATE TABLE
yugabyte=# select Id from FoO; -- any capitalization works if unquoted
 id
----
(0 rows)

yugabyte=# create table "bAr"("iD" int PRIMARY KEY);
CREATE TABLE
yugabyte=# select iD from bAr; -- no capitalization works because identifiers will let lowercased if unquoted
ERROR:  relation "bar" does not exist
LINE 1: select iD from bAr;
                       ^
yugabyte=# select "iD" from "bAr"; -- quoting with the same as the declared capitalization will work.
 iD
----
(0 rows)

yugabyte=# \d
        List of relations
 Schema | Name | Type  |  Owner
--------+------+-------+----------
 public | bAr  | table | yugabyte
 public | foo  | table | yugabyte
(2 rows)
```

Reply:
Fixed this section